Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -20,28 +20,23 @@ def change_rpath(file, old_prefix, new_prefix)
 
     patchelf = DevelopmentTools.locate "patchelf"
     odie "Could not locate patchelf, please: brew install patchelf." if patchelf.nil?
+    cmd = [patchelf]
 
-    cmd_rpath = [patchelf, "--print-rpath", file]
-    old_rpath = Utils.popen_read(*cmd_rpath, err: :out).strip
+    old_rpath = file.rpath
+    new_rpath = if old_rpath
+      rpath = old_rpath
+              .split(":")
```

Comment:
```suggestion
      rpath = old_rpath.split(":")
```

and same on lines below.

Reply:
@MikeMcQuaid  i did the requested change, putting all in one line exceeds line limit.
and it looks odd now.